app/contributing/index.md

```diff
@@ -0,0 +1,680 @@
+          You can create multiple Dev Portals based on selected APIs with appropriate visibility and access control settings.
+        icon: /assets/icons/dev-portal.svg
+        cta:
+          url: "https://cloud.konghq.com/portals/create"
+```
+<!--vale on-->
+
+### Plugin cards
+
+Displays the title, icon, description, and link to a plugin as a card/tile on a landing page.
+
+<!--vale off-->
+```yaml
+- blocks:
+    - type: plugin
+      config:
+        slug: rate-limiting
+```
+<!--vale on-->
+
+### Autogenerated how to list
+
+Displays a list of how to pages that contain the specified metadata.
+
+<!--vale off-->
+```yaml
+-
+  blocks:
+    - type: how_to_list
+      config:
+        tags:
+          - rate-limiting
+        quantity: 5
+```
+<!--vale on-->
+
+### Related resources list
+
+Displays a list of related resources.
+
+<!--vale off-->
+```yaml
+-
+  blocks:
+    - type: related_resources
+      config:
+        - text: Protecting Services With {{site.base_gateway}} Rate Limiting
+          type: blog
+          url: https://konghq.com/blog/engineering/kong-gateway-rate-limiting
+```
+<!--vale on-->
+
+### Feature table
+
+Displays a feature table that shows if features are supported or unsupported.
+
+<!--vale off-->
+```yaml
+- blocks:
+          - type: feature_table
+            config:
+              columns:
+                - title: Rate Limiting
+                  key: oss
+                - title: Rate Limiting Advanced
+                  key: ee
+              features:
+                - title: Rate limit based on `consumer`, `consumer-group`, `credential`, `ip` and `service`
+                  oss: true
+                  ee: true
+```
+<!--vale on-->
+
+### Table
+
+Displays a regular table that contains text.
+
+<!--vale off-->
+```yaml
+- blocks:
+    - type: table
+      config:
+        columns:
+          - title: Authentication flow or grant
+            key: auth
+          - title: Description
+            key: description
+          - title: Plugin example configuration
+            key: example
+          - title: How-to guide
+            key: howto
+        rows:
+          - auth: Authorization Code flow
+            description: |
+              In an Authorization Code flow, clients exchange an authorization code for an access token.
+              <br><br>
+              [Workflow diagram &rarr;](/plugins/openid-connect/#authorization-code-flow)
+            example: "[Authorization Code flow example](/plugins/openid-connect/examples/authorization-code/)"
+            howto: "[OpenID Connect and Keycloak with the auth code flow](/how-to/configure-oidc-with-auth-code-flow/)"
+```
+<!--vale on-->
+
+## Tools
+
+The Kong Developer site uses tools to help with automation.
+
+### Broken link checker: ignore URls
+
+<!--vale off-->
+```
+tools/broken-link-checker/config/ignored_targets.json
+```
+<!--vale on-->
+
+### Generate kong.conf reference
```

Comment:
```suggestion
```
Maybe take this out for now, and we can add generation guides in a follow-up? Fabian has readmes for all of them - could also just link to those.